Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -2403,3 +2395,19 @@ callDriver state callType = do
         let _ = unsafePerformEffect $ logEventWithTwoParams state.data.logField ("ny_user_"<> callType <>"_call_click") "trip_id" (state.props.bookingId) "user_id" (getValueToLocalStore CUSTOMER_ID)
         pure NoAction
     ]
+
+getNightChargeInfo :: Array EstimateAPIEntity -> {nightShiftMultiplier :: Number, nightCharges :: Boolean}
+getNightChargeInfo estimates =
```

Comment:
since type are same for both cases, make return type of this function with almost all of the fields and we can use only fields which are required

Reply:
like it should return `pickupCharges`, `additionalCharges` ... which can we used by by single estimate flow for rate card